pkgs/development/python-modules/gps3/default.nix

```
@@ -5,13 +5,13 @@
 
 buildPythonPackage rec {
   pname = "gps3";
-  version = "0.33.3";
+  version = "2017-11-01";
```

```suggestion
  version = "unstable-2017-11-01";
```
please add a prefix.

---

done